squash_bot/match_tracker/formatters.py

```diff
@@ -13,6 +13,17 @@ class Formatter(abc.ABC):
     def format_matches(cls, matches: dataclasses.Matches) -> str: ...
 
 
+class BasicFormatter(Formatter):
+    @classmethod
+    def format_matches(cls, matches: dataclasses.Matches) -> str:
+        inner_message = ""
+        for match in matches.match_results:
+            match_string = _match_string(match)
+            inner_message += f"\n{match_string}"
+
+        return f"```{inner_message}```"
```

Comment:
Might be nice to have some form of static method in the framework to do some equivalent of `ToCodeBlock()` to avoid having to add the '`' all the time

Reply:
Makes sense -- we could have a subclass of formatter that handles this:

```py
class CodeBlockFormatter(Formatter):
    def format(...):
        return f"```{self.inner_message(...)}```"

    def inner_message(...):
        ...
```

or maybe that's too try-hard, and we could just have a function to do it 

```py
  def format(...):
    ...
    return _to_code_block(inner_message)

def _to_code_block(inner_message: str) -> str:
    return f```{inner_message}```
```